gluten-core/src/main/scala/io/glutenproject/expression/TernaryExpressionTransformer.scala

```diff
@@ -108,6 +108,38 @@ case class StringLocateTransformer(
   }
 }
 
+case class StringSplitTransformer(
+    substraitExprName: String,
+    srcExpr: ExpressionTransformer,
+    regexExpr: ExpressionTransformer,
+    limitExpr: ExpressionTransformer,
+    original: StringSplit)
+  extends ExpressionTransformer
+  with Logging {
+
+  override def doTransform(args: java.lang.Object): ExpressionNode = {
+    // In velox, split function just support tow args, not support limit arg for now
+    if (!BackendsApiManager.isVeloxBackend) {
+      return TernaryExpressionTransformer(
+        substraitExprName,
+        srcExpr,
+        regexExpr,
```

Comment:
We only support const char in velox, thus it alse need a check.

Reply:
Thanks for your review. I already add this constraint when pattern string length gt 1 then fallback on commit cbe5c6.
In the future,  we will support string pattern in velox.